### PR TITLE
Fixed #31909 -- Fixed typo in docs/ref/contrib/admin/index.txt.

### PR DESCRIPTION
**Pull request by endersonmenezes**
This is my first PR here, I read some examples of people who helped correct the documentation before, I hope I am not doing anything wrong. I encountered this error when I was looking for solutions to a Stackoverflow issue.

https://github.com/django/django/pull/12159

### CODE REVIEW

**Review by knyghty — APPROVED** [review]
Thanks, looks right to me 👍

File: docs/ref/contrib/admin/index.txt [hunk 1]
```diff
@@ -35,7 +35,7 @@ If you're not using the default project template, here are the requirements:
 #. Configure a :class:`~django.template.backends.django.DjangoTemplates`
    backend in your :setting:`TEMPLATES` setting with
    ``django.contrib.auth.context_processors.auth``,
-   ``django.contrib.messages.context_processors.request``, and
+   ``django.template.context_processors.request``, and
```
Comment (felixxm) [inline docs/ref/contrib/admin/index.txt:38]: Please fix it also in the `versionchanged` annotation, line 45.

Comment (endersonmenezes) [reply docs/ref/contrib/admin/index.txt:38]: Just below, and I forgot ^^

**Comment by felixxm** [comment]
@endersonmenezes Thanks :+1: Welcome aboard :boat:

**Comment by endersonmenezes** [comment]
It was very simple, but I already felt useful. I will be researching things at my level to complement :)

**Pull request merged**